pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControlUtils.java

```diff
@@ -93,4 +93,17 @@ public void validatePermission(Optional<String> tableNameOpt, AccessType accessT
           Response.Status.FORBIDDEN);
     }
   }
+
+
+  public void validatePermission(HttpHeaders httpHeaders, String endpointUrl,
+      AccessControl accessControl) {
+    boolean hasPermission;
+    String accessTypeToEndpintMsg =
+        String.format("request to the endpioint '%s'", endpointUrl);
```

Comment:
delete `accessTypeToEndpintMsg` and merge if & declaration of `hasPermission`?

Reply:
You are right. I have accepted your advice. It will make this method clearer. Please review it again.